### PR TITLE
Added Roslyn CSharp code formatting options to omnisharp.json.

### PR DESCRIPTION
**Pull request by bstockus**
Addresses Issue #31

### CODE REVIEW

**Comment by david-driscoll** [comment]
@DustinCampbell thoughts?

Looks valuable to me.

File: src/OmniSharp.Roslyn.CSharp/Workers/Formating/FormattingWorker.cs [hunk 1]
```diff
@@ -94,5 +141,29 @@ public static SyntaxNode FindFormatTarget(SyntaxNode root, int position)
 
             return await LinePositionSpanTextChange.Convert(document, textChanges);
         }
+
+        private static LabelPositionOptions LabelPositionOptionForStringValue(string value)
+        {
+            switch (value) {
+                case "leftMost":
+                    return LabelPositionOptions.LeftMost;
+                case "noIndent":
```
Comment (david-driscoll) [inline src/OmniSharp.Roslyn.CSharp/Workers/Formating/FormattingWorker.cs]: That's not a bad idea.

**Comment by DustinCampbell** [comment]
Hey @bstockus: Are you OK with matching value case-insensitively?

**Comment by bstockus** [comment]
That makes sense to me.  I will add a commit with those changes to this PR.

**Comment by david-driscoll** [comment]
LGTM

**Pull request merged**